resize.go

```diff
@@ -9,12 +9,14 @@ import "C"
 import (
 	"errors"
 	"math"
+  "runtime"
 )
 
 // Resize is used to transform a given image as byte buffer
 // with the passed options.
 func Resize(buf []byte, o Options) ([]byte, error) {
 	defer C.vips_thread_shutdown()
+  defer runtime.KeepAlive(buf)
```

Comment:
At first glance I think these defer statements should be in a different order. Defer statements are invoked in reverse order, this means that `runtime.KeepAlive(buf)` is run _before_ `C.vips_thread_shutdown()`, is that intended?

Also: The indentation seems wrong